build_posix/configure.ac.in

```diff
@@ -64,6 +64,10 @@ AS_CASE([$host_cpu],
 	[s390x*], [wt_cv_zseries="yes"],
 	[wt_cv_zseries="no"])
 AM_CONDITIONAL([ZSERIES_HOST], [test "$wt_cv_zseries" = "yes"])
+AS_CASE([$host_cpu],
```

Comment:
Pure copy paste from the nearby arm stanza, but I did an autoreconf and configure / make and it all seemed to work OK.